Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get -y update && apt-get -y upgrade && \
   apt-get -y update --fix-missing && \
   apt-get -y install && apt-get -y upgrade && \
   apt-get -y install software-properties-common && \
+  apt-get -y install curl && \
```

Comment:
these are just for testing right?

Now I know why I didn't need to install curl myself yesterday :P

Reply:
No, these are being used in the liveness script, which I've actually added to my PR now 😅